sorts/merge_sort.py

```diff
@@ -51,6 +51,10 @@ def _merge():
     import doctest
 
     doctest.testmod()
-    user_input = input("Enter numbers separated by a comma:\n").strip()
-    unsorted = [int(item) for item in user_input.split(",")]
-    print(*merge_sort(unsorted), sep=",")
+    try:
+        user_input = input("Enter numbers separated by a comma:\n").strip()
+        unsorted = [int(item) for item in user_input.split(",")]
+        sorted_list = merge_sort(unsorted)
+        print(*sorted_list, sep=",")
+    except ValueError:
+        print("Invalid input. Please enter valid integers separated by commas.")
```

Comment:
https://peps.python.org/pep-0008/#programming-recommendations
> Limit the try clause to the absolute minimum amount of code necessary.

```suggestion
    try:
        user_input = input("Enter numbers separated by a comma:\n").strip()
    except ValueError:
        print("Invalid input. Please enter valid integers separated by commas.")
    else:
        unsorted = [int(item) for item in user_input.split(",")]
        print(f"{merge_sort(unsorted) = }")    
```